public/locales/en/translation.json

```diff
@@ -198,7 +198,11 @@
     "ssn-error": "Please check your SSN.",
     "invalid-user-error": "We couldn't find you in the system.",
     "invalid-recaptcha-error": "Human verification expired. Please check the checkbox again.",
-    "session-timed-out": "Logged out after being idle for 30 minutes."
+    "session-timed-out": "Logged out after being idle for 30 minutes.",
```

Comment:
Can you also update the invalid-recaptcha-error and session-time-out messages? Meghana provided some text on issue #281.